code/lib/csf-tools/src/CsfFile.ts

```diff
@@ -543,6 +547,13 @@ export const formatCsf = (csf: CsfFile, options: FormatOptions = { sourceMaps: f
   return code;
 };
 
+/**
+ * Use this function, if you want to preserve styles. Uses recast under the hood.
+ */
+export const printCsf = (csf: CsfFile, options: Options = {}) => {
+  return recast.print(csf._ast, options);
```

Comment:
As you pointed out, last week 😅 , this is breaking change.

For 2 reasons:
* recast has different options than babel, especially related to sourceMaps. I'm not sure how to refactor the csf-plugin (that uses options.sourceMaps) so that it keeps working. (But I can investigate)
* We only return the `result` now when options.sourceMaps is set, so always returning `result` is a breaking change. I can refactor our own usages, to get `result.code` when needed, but I'm not sure about what all the other usages are (eslint plugin maybe)

Therefore @Integrayshaun and I thought it would be better to just add a new method and not potentially break unexpected stuff.

Reply:
Gotcha. Thanks for humoring me! 😂